### PR TITLE
Unicode15

### PR DESCRIPTION
**Pull request by xadxura**
PR to merge staged changes for Unicode 15 into main.

### CODE REVIEW

File: USE/IndicShapingInvalidCluster.txt [hunk 1]
```diff
@@ -8,98 +8,106 @@
 #
 # Scope: This file enumerates sequences of characters that should be treated as invalid clusters
 
-  0905 0946       ; # DEVANAGARI LETTER A, DEVANAGARI VOWEL SIGN SHORT E
```
Comment (xadxura) [inline USE/IndicShapingInvalidCluster.txt:11]: This is mostly whitespace changes to allow width for Khojki sequence with three code points.

**Review by xadxura — COMMENTED** [review]
Updates for Unicode 15 and bug fixes.

File: USE/IndicPositionalCategory-Additional.txt [hunk 2]
```diff
@@ -22,6 +22,8 @@ A9BF          ; Bottom  # Mc      JAVANESE CONSONANT SIGN CAKRA
 11127..11129  ; Bottom  # Mn  [3] CHAKMA VOWEL SIGN A..CHAKMA VOWEL SIGN II
 1112D         ; Bottom  # Mn      CHAKMA VOWEL SIGN AI
 11130         ; Bottom  # Mn      CHAKMA VOWEL SIGN OI
+1BF2..1BF3    ; Nukta   # Mc  [2] BATAK PANGOLAT..BATAK PANONGONAN # see USE issue #20
```
Comment (NorbertLindenberg) [inline USE/IndicPositionalCategory-Additional.txt]: Issue #20 proposed Bottom, not Nukta, for the positional category.

**Comment by dscorbett** [comment]
Because U+13440 and U+13447..U+13455 are overridden to Indic_Syllabic_Category=Cantillation_Mark, they should be given Indic_Positional_Category values.

**Comment by xadxura** [comment]
> Because U+13440 and U+13447..U+13455 are overridden to Indic_Syllabic_Category=Cantillation_Mark, they should be given Indic_Positional_Category values.

Added IPC assignments for Hieroglyph non-spacing marks.

**Pull request merged**